includes/class-wc-calypso-bridge-woocommerce-admin-features.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Control wc-admin features in the eCommerce Plan.
+ *
+ * @package WC_Calypso_Bridge/Classes
+ * @since   1.3.0
+ * @version 1.0.0
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC EComm Bridge
+ */
+class WC_Calypso_Bridge_WooCommerce_Admin_Features {
+
+	/**
+	 * Class Instance.
+	 *
+	 * @var WC_Calypso_Bridge_WooCommerce_Admin_Features instance
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		add_action( 'plugins_loaded', array( $this, 'initialize' ), 2 );
+	}
+
+	/**
+	 * Add hooks and filters if WooCommerce is active.
+	 */
+	public function initialize() {
+		if ( ! function_exists( 'WC' ) ) {
+			return;
+		}
+
+		add_filter( 'woocommerce_admin_features', array( $this, 'filter_wc_admin_enabled_features' ) );
+	}
+
+	/**
+	 * Set feature flags for WooCommerce Admin front end at run time.
+	 *
+	 * @param array $features List of features 'feature' => bool indicating whether the feature is enabled.
+	 * @return array
+	 */
+	public function filter_wc_admin_enabled_features( $features ) {
+		if ( ! array_key_exists( 'remote-inbox-notifications', $features ) ) {
+			$features[] = 'remote-inbox-notifications';
```

Comment:
If this is `'feature' => bool` format, shouldn't this be:

```suggestion
			$features['remote-inbox-notifications'] = true;
```

Reply:
Sorry for the slow response here @joshuatf , if I `error_log()` the $features that are passed in by the filter call, it looks like it is a flat array:

```
[20-Aug-2020 02:11:58 UTC] Array
(
    [0] => activity-panels
    [1] => analytics
    [2] => analytics-dashboard
    [3] => analytics-dashboard/customizable
    [4] => coupons
    [5] => marketing
    [6] => onboarding
    [7] => shipping-label-banner
    [8] => store-alerts
    [9] => wcpay
    [10] => homescreen
)
```

Which I believe jives with how we set the array in wc-admin [here](https://github.com/woocommerce/woocommerce-admin/blob/main/src/FeaturePlugin.php#L275) which uses array_keys

Also yes it seems those zip files in pastebin have gone missing. [Here is a google drive folder](https://drive.google.com/drive/folders/1dEKIYcaXZ28Rz8nl7qruQ7kBTIRYwAMb?usp=sharing) with the zips.

Though with 1.4/4.4 out the door, I think we can use the new ecommerce plan rule on that inbox notif now. But we can keep the testing instructions the same here for now and circle back on that update to the notif feed later.

Reply:
Gotcha.  This looks good then; I think we can update the above doc block and remove the above portion about `'feature' => bool` to avoid any confusion.